### ISSUE TITLE
Moving cursor via j/k does not retain column

### ISSUE THREAD

**Issue by jordwalke** [sublimehq/Vintage#8]
In any decent editor, moving up and down, in any mode will preserve the column. Even in sublime (non-vintage mode) the column is preserved. Do you know a way to fix the keymapping so that this is the case? The problem may be in the implementation of the command itself.

**Comment by omarramos** [comment]
+1!

**Comment by jtinsky** [comment]
+1

**Comment by jordwalke** [comment]
Has anyone found a fix to this? Unfortunately, this is the only thing holding me back from using sublime. Otherwise, Vintage mode is a pretty faithful recreation of Vim mode.
I tried looking into the mappings and I had absolutely no luck. I think someone needs to dig into the python code.

**Comment by jordwalke** [comment]
I was thinking a quick hack to unblock this could be to remap j/k in command mode to (<Escapce>, <Up>, <Insert> , <Right>). Does anyone know how to remap a key to a sequence of other keys?

**Comment by jordwalke** [comment]
Here's a quick fix to put in your personal keybindings. The only difference between this and vim, is that when moving down or up, it will sometimes place the cursor one character past the end of line. If anyone knows how to resolve that please let me know, but to be honest you could imagine a world where vim itself did that.

```
    { "keys": ["j"], "command": "move",
   "args": {"by": "lines", "forward": true, "extend": false},
   "context": [
     {"key": "setting.command_mode"},
   {"key": "selection_empty", "operator": "equal", "operand": true, "match_all": false },
   { "key": "setting.vintage_ctrl_keys" }

   ]
},
{ "keys": ["k"], "command": "move",
   "args": {"by": "lines", "forward": false, "extend": false},
   "context": [
     {"key": "setting.command_mode"},
   {"key": "selection_empty", "operator": "equal", "operand": true, "match_all": false },
     { "key": "setting.vintage_ctrl_keys" }

   ]
},
{ "keys": ["j"], "command": "move",
   "args": {"by": "lines", "forward": true, "extend": true},
   "context": [
     {"key": "setting.command_mode"},
   { "key": "num_selections", "operand": 1},
   {"key": "selection_empty", "operator": "equal", "operand": false, "match_all": false }
   ]
},
{ "keys": ["k"], "command": "move",
   "args": {"by": "lines", "forward": false, "extend": true},
   "context": [
     {"key": "setting.command_mode"},
   { "key": "num_selections", "operand": 1},
   {"key": "selection_empty", "operator": "equal", "operand": false, "match_all": false }
   ]
},
```

**Comment by omarramos** [comment]
awesome, thanks for sharing!

**Comment by guillermooo** [comment]
A different approach to solve this can be found at #27.

**Comment by guillermooo** [comment]
Fixed at 3e769328701a10e62a19dad8c2c380cbe9bbec43

**Comment by jordwalke** [comment]
Thanks!
